src/main/scala/fm/sbt/S3URLHandler.scala

```diff
@@ -297,12 +297,15 @@ final class S3URLHandler extends URLHandler {
       } yield new EndpointConfiguration(serviceEndpoint, signingRegion)
 
       // Path Style Access is deprecated by Amazon S3 but LocalStack seems to want to use it
-      val pathStyleAccess: Boolean = Option(System.getenv("S3_PATH_STYLE_ACCESS")).map{ _.toBoolean }.getOrElse(false)
+      val pathStyleAccess: Boolean = Option(System.getenv("S3_PATH_STYLE_ACCESS")).exists(_.toBoolean)
+
+      // Rerouting can cause replacing the user custom endpoint with the S3 default one (s3.amazonaws.com). Default is true
+      val forceGlobalBucketAccessEnabled: Boolean = Option(System.getenv("S3_FORCE_GLOBAL_BUCKET_ACCESS")).forall(_.toBoolean)
```

Comment:
I wonder if it is better to keep this defaulting to true to avoid breaking anybody currently relying on this behavior.

Reply:
It's true by default. `forall` returns `true` when `Option` is empty.